install/0000_80_machine-config-operator_07_etcdquorumguard_deployment.yaml

```diff
@@ -37,21 +37,14 @@ spec:
       - key: node-role.kubernetes.io/master
         effect: NoSchedule
         operator: Exists
-      - key: node.kubernetes.io/memory-pressure
-        effect: NoSchedule
-        operator: Exists
-      - key: node.kubernetes.io/disk-pressure
-        effect: NoSchedule
-        operator: Exists
       - key: node.kubernetes.io/not-ready
         effect: NoExecute
         operator: Exists
+        tolerationSeconds: 120
       - key: node.kubernetes.io/unreachable
         effect: NoExecute
         operator: Exists
-      - key: node.kubernetes.io/unschedulable
-        effect: NoExecute
-        operator: Exists
+        tolerationSeconds: 120
```

Comment:
I believe this needs to tolerate unschedulable permanently; this needs to remain around while etcd itself is.

Also, I have another PR open on this area (#688); perhaps whatever we decide to do should get merged?

Reply:
I closed #688 vice this.